11.md

```diff
@@ -35,6 +37,14 @@ A relay may select a `name` for use in client software.  This is a string, and S
 
 Detailed plain-text information about the relay may be contained in the `description` string.  It is recommended that this contain no markup, formatting or line breaks for word wrapping, and simply use double newline characters to separate paragraphs.  There are no limitations on length.
 
+### Banner
+
+If nostr relay management is not to be relegated to PhD or protocol developers only, an effort should be made by relay owners to communicate with non-dev non-technical nostr end users. A banner is a visual representation of the relay. It should aim to visually communicate the brand of the relay, complementing the text `Description`. [Here is an example banner](https://image.nostr.build/232ddf6846e8aea5a61abcd70f9222ab521f711aa545b7ab02e430248fa3a249.png) mockup as visualized in Damus iOS relay view of the Damus relay.
+
+### Icon
+
+Icon is a compact visual representation of the relay for use in UI with limited real estate such as a nostr user's relay list view. Example relay icon link `https://damus.io/favicon.ico`.
```

Comment:
I would not use `.ico` here, `.png` is probably a better example, but it could be any image file.

Reply:
Kindly provide the example you would like included.

Reply:
updated